src/sonic-config-engine/minigraph.py

```diff
@@ -1944,6 +1944,9 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
             port['mux_cable'] = "true"
 
     if static_routes:
+        # Enable static Route BFD by default for static route in chassis-packet
+        if switch_type == "chassis-packet":
+            static_routes['bfd'] = "true"
```

Comment:
UT failed. 
It can pass UT test_minigraph_bgp_packet_chassis_static_route if changed to the following code
```
        # Enable static Route BFD by default for static route in chassis-packet
        if switch_type == "chassis-packet":
            for static_route in static_routes:
                static_route['bfd'] = "true"
```